other/materials_designer/add_vacancy_defect.ipynb

```diff
@@ -0,0 +1,220 @@
+{
```

Comment:
<p>Create point defects in a bulk material should be the name of the notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>